discovery-infra/log_scrap.py

```diff
@@ -103,6 +104,33 @@ def elastefy_events(self, cluster, event_list):
         cluster_bash_data = process_metadata(metadata_json)
         event_names = get_cluster_object_names(cluster_bash_data)
 
+        self.process_and_log_events(cluster_bash_data, event_list, event_names)
+
+
+        if self.dose_cluster_needs_full_update(cluster_id, event_list):
+            log.info(f"Cluster {cluster_id} logged events are not same as the event count, logging all clusters events")
```

Comment:
please add the number of events that are missing to the log

Reply:
why?

Reply:
added on 126